[ASM] Expander NGFWv1 (#29831)
demisto/content: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_0.md

#### Playbooks

##### Cortex ASM - Remediation

> - Updated the playbook to include the new **Cortex ASM - On Prem Remediation** playbook.

Reviewer: ```suggestion
- Updated the playbook to include the new **Cortex ASM - On Prem Remediation** playbook.
```
```suggestion
Updated the playbook to include the new **Cortex ASM - On Prem Remediation** playbook.
```

##### New: Cortex ASM - On Prem Remediation

> - Added the **Cortex ASM - On Prem Remediation** playbook that adds new block rule(s) to on-prem firewall vendors in order to block internet access for exposures.

Reviewer: ```suggestion
Added the **Cortex ASM - On Prem Remediation** playbook that adds new block rule(s) to on-prem firewall vendors in order to block internet access for exposures.
```

##### Cortex ASM - Remediation Path Rules

> - Updated the playbook to include the new on-prem remediation options.

Reviewer: ```suggestion
Updated the playbook to include the new on-prem remediation options.
```

##### Cortex ASM - Remediation Objectives

> - Updated the playbook to include the new on-prem remediation options.

Reviewer: ```suggestion
Updated the playbook to include the new on-prem remediation options.
```

##### Cortex ASM - Enrichment

> - Updated the playbook to include the new **Cortex ASM - On Prem Enrichment** playbook.

Reviewer: ```suggestion
Updated the playbook to include the new **Cortex ASM - On Prem Enrichment** playbook.
```

##### New: Cortex ASM - On Prem Enrichment

> - Added the **Cortex ASM - On Prem Enrichment** playbook that given and IP address, port and protocol of a service, enriches using on-prem integrations to find the related firewall rule and other related information. 

Reviewer: ```suggestion
Added the **Cortex ASM - On Prem Enrichment** playbook that given an IP address, port, and protocol of a service, enriches using on-prem integrations to find the related firewall rule and other related information. 
```

##### Cortex ASM - Azure Enrichment

> - Fixed an issue with conditional not have 'else' statement.

Reviewer: ```suggestion
Fixed an issue with the condition not have an 'else' statement.
```
